### PR TITLE
add getPeaks background compatibility

### PR DESCRIPTION
**Pull request by entonbiba**
Fix to allow old style getPeaks(length) usage with new getPeaks(length,first,last).
If no `first` and `last` parameter is set it will set the default `0` and `length-1`.

#1067

### CODE REVIEW

File: src/webaudio.js [hunk 1]
```diff
@@ -218,6 +218,9 @@ WaveSurfer.WebAudio = {
     getPeaks: function (length, first, last) {
         if (this.peaks) { return this.peaks; }
 
+        var first = first || 0;
```
Comment (katspaugh) [inline src/webaudio.js]: No need for `var`, since it's already declared as an argument.

File: src/webaudio.js [hunk 2]
```diff
@@ -218,6 +218,9 @@ WaveSurfer.WebAudio = {
...
+        var last = last || length-1;
```
Comment (katspaugh) [inline src/webaudio.js]: `last = last || length - 1;` (no `var` and spacing around the minus)

**Comment by entonbiba** [comment]
@katspaugh updated :)

**Comment by katspaugh** [comment]
Thanks! 👍

**Pull request merged**